.github/stale.yml

```diff
@@ -0,0 +1,36 @@
+# Configuration for Probot (Stale) - https://github.com/probot/stale
+
+# Ignore issues in a project (defaults to false)
+exemptProjects: true
+
+# Ignore issues in a milestone (defaults to false)
+exemptMilestones: true
+
+# Label to use when marking as stale
+staleLabel: archived
+
+# Limit the number of actions to 1 per hour
+limitPerRun: 1
+
+# Do not archive issues that require a SEMVER major release, those normally
+# have a longer lifeycle.
+exemptLabels:
+  - SEMVER-MAJOR
+
+pulls:
+  daysUntilStale: 60
+  daysUntilClose: 5
```

Comment:
What if we start less aggressively by marking as stale the issues that are much older than this, like a year old or more? I think that would quickly get things under control so we can treat the remaining issues with more care.

Reply:
The initial value is somewhat arbitrary - 60 days for PRs worked well for Maps SDK. I suggest we start with that value to be consistent with that experience, and adjust as needed as we learn more. I've set `limitPerRun` to 1 precisely with that in mind. The bot will be slow moving :)